generate.py

```diff
@@ -112,29 +112,30 @@ def create_cmakelists(filepath, project_name, enabled_libraries):
 
         f.write("\n")
         f.write('# ===============================================\n')
-        f.write('# Compile flags\n')
+        f.write('# compiler flags\n')
         f.write('if (MSVC)\n')
         f.write('    target_compile_options(${PROJECT_NAME} PUBLIC\n')
         f.write('        /MP\n')
         f.write('    )\n')
         f.write('else()\n')
         f.write('    target_compile_options(${PROJECT_NAME} PUBLIC\n')
         f.write('        -Wall\n')
-        f.write('        -Werror\n')
+        f.write('        -Wno-unused-variable\n')
```

Comment:
I'm ok with removing -Wall, but -Wno-unused-variable is a rather personal choice ;)